elasticjob-example-lite-java is run error

ElasticJob-Lite 
 

### Actual behavior

Run : org.apache.shardingsphere.elasticjob.lite.example.JavaMain 


```
[ERROR] 2020-10-30 13:55:23,520 --main-- [org.apache.zookeeper.server.NIOServerCnxnFactory] Thread Thread[main,5,main] died 
java.lang.NullPointerException: null
	at java.util.concurrent.ConcurrentHashMap.get(ConcurrentHashMap.java:936) ~[na:1.8.0_231]
	at org.apache.shardingsphere.elasticjob.infra.spi.ElasticJobServiceLoader.lambda$newTypedServiceInstance$4(ElasticJobServiceLoader.java:81) ~[classes/:na]
	at java.util.Optional.map(Optional.java:215) ~[na:1.8.0_231]
	at org.apache.shardingsphere.elasticjob.infra.spi.ElasticJobServiceLoader.newTypedServiceInstance(ElasticJobServiceLoader.java:81) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.JobScheduler.validateJobErrorHandlerProperties(JobScheduler.java:122) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.JobScheduler.validateJobProperties(JobScheduler.java:118) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.JobScheduler.<init>(JobScheduler.java:102) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.api.bootstrap.impl.ScheduleJobBootstrap.<init>(ScheduleJobBootstrap.java:40) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.example.JavaMain.setUpHttpJob(JavaMain.java:101) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.example.JavaMain.main(JavaMain.java:71) ~[classes/:na]

````


### Reason analyze (If you can)

jobErrorHandlerType  is null , need default to be "LOG".


```java
private void validateJobErrorHandlerProperties() {
        ElasticJobServiceLoader.newTypedServiceInstance(JobErrorHandlerPropertiesValidator.class, jobConfig.getJobErrorHandlerType(), jobConfig.getProps())
                .ifPresent(validator -> validator.validate(jobConfig.getProps()));
    }
```

--- a/elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/internal/schedule/JobScheduler.java
+++ b/elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/internal/schedule/JobScheduler.java
@@ -119,8 +119,10 @@ public final class JobScheduler {
     }
     
     private void validateJobErrorHandlerProperties() {
-        ElasticJobServiceLoader.newTypedServiceInstance(JobErrorHandlerPropertiesValidator.class, jobConfig.getJobErrorHandlerType(), jobConfig.getProps())
-                .ifPresent(validator -> validator.validate(jobConfig.getProps()));
+        if (null != jobConfig.getJobErrorHandlerType()) {
+            ElasticJobServiceLoader.newTypedServiceInstance(JobErrorHandlerPropertiesValidator.class, jobConfig.getJobErrorHandlerType(), jobConfig.getProps())
+                    .ifPresent(validator -> validator.validate(jobConfig.getProps()));
+        }
     }
     
     private void setGuaranteeServiceForElasticJobListeners(final CoordinatorRegistryCenter regCenter, final Collection<ElasticJobListener> elasticJobListeners) {

